lib/classes/Swift/Plugins/DecoratorPlugin.php

```diff
@@ -157,12 +157,12 @@ public function getReplacementsFor($address)
     {
         if ($this->_replacements instanceof Swift_Plugins_Decorator_Replacements) {
             return $this->_replacements->getReplacementsFor($address);
-        } else {
-            return isset($this->_replacements[$address])
+        }
+
+        return isset($this->_replacements[$address])
                 ? $this->_replacements[$address]
                 : null
```

Comment:
Certainly! There are many other locations in the code with that kind of weird indentations.